conf/machine/raspberrypi3.conf

```diff
@@ -13,6 +13,8 @@ MACHINE_EXTRA_RRECOMMENDS += "\
     bluez-firmware-rpidistro-bcm4345c0-hcd \
 "
 
+MACHINE_FEATURES += "vc4graphics"
```

Comment:
this change should be make common across all rpi machine, rest of the change should be dropped from this patch.

Reply:
Hi @kraj ,
Committed as per the change, Kindly review,
9cd3dc794461ccfb6c1354d16a8fc7deabe53c3d